### ISSUE TITLE
Sporadic "Not Found" errors

### ISSUE THREAD

**Issue by DonMcNamara** [softprops/action-gh-release#373]
I'm seeing sporadic failures of this action with a message of "not found". It seems to eventually work on retry.

I'm wondering if this might be an eventual consistency issue with github. I have no idea, that is just a guess.

Is anyone else seeing this behavior?

```
2023-07-10T20:34:32.8261356Z ♻️ Deleting previously uploaded asset builder-debug.yml...
2023-07-10T20:34:32.8263347Z ♻️ Deleting previously uploaded asset latest-linux.yml...
2023-07-10T20:34:32.9152383Z ⬆️ Uploading my-app-2.0.23-x64...
2023-07-10T20:34:33.2576571Z ##[error]Not Found
```

I didn't see it in the readme, but any chance there is a verbose option to troubleshoot this kind of thing?

**Comment by DonMcNamara** [comment]
Update: I think this is being caused by multiple concurrent deletes executing across a matrix of runners. The upload/delete functions are executed in parallel so the log lines are being interleaved.

**Comment by RainbowMango** [comment]
I met this issue as well.
https://github.com/karmada-io/karmada/actions/runs/5748249031/job/15580915704

**Comment by Virtlink** [comment]
I've seen this happen when more than one file has the same name and is being uploaded to the GitHub release. The error message could be better though.

**Comment by mitchelsellers** [comment]
I too am seeing this error and it is very sporadic

**Comment by chenrui333** [comment]
I could not reproduce or isolate this on current master and the latest 2.5.2 release in the current regression sweep, so I am closing it to keep the backlog focused on confirmed current bugs. If you are still seeing a different current failure, please open a new issue with a minimal repro or sanitized workflow snippet and the failing logs.
